proto/rill/runtime/v1/api.proto

```diff
@@ -119,6 +120,20 @@ service RuntimeService {
     };
   }
 
+  rpc GenerateResolver(GenerateResolverRequest) returns (GenerateResolverResponse) {
+    option (google.api.http) = {
+      post: "/v1/instances/{instance_id}/files/generate-resolver",
+      body: "*"
+    };
+  }
+
+  rpc GenerateChartSpec(GenerateChartSpecRequest) returns (GenerateChartSpecResponse) {
+    option (google.api.http) = {
+      post: "/v1/instances/{instance_id}/files/generate-chart",
+      body: "*"
+    };
+  }
```

Comment:
1. Add a docstring for each API (can be short, like for the other APIs) :) 
2. Since these endpoints don't generate files, they shouldn't use the `/files`/ path. For consistency between all the generator APIs, you might also consider refactoring `GenerateMetricsViewFile` to `GenerateMetricsView` and have the frontend write the file.

Reply:
for 2 will raise a separate PR to refactor `GenerateMetricsViewFile`. This PR is already getting large.